.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
Whenever you open a live server, don't open multiple HTML files on the server. Since the previous one was already running on port 3000, it had to create another one and thus this got on to settings.json. Basically, this is avoidable.

Reply:
Okay will take care of this.